packages/webapp/src/containers/SensorReadings/saga.js

```diff
@@ -161,7 +161,9 @@ export function* getSensorsReadingsSaga({ payload }) {
             if (acc[currentValueUnixTime]) {
               acc[currentValueUnixTime] = {
                 ...acc[currentValueUnixTime],
-                [`${readings.stationName}`]: cv?.main?.temp,
+                [`${i18n.t('SENSOR.TEMPERATURE_READINGS_OF_SENSOR.AMBIENT_TEMPERATURE_FOR')} ${
```

Comment:
@Duncan-Brain I'm not sure -- I re-introduced because I was following the Figma file -- but I figured it should be run by Kevin + Caro before deciding one way or another.